### PR TITLE
Update edit profile form field validation and password visibility

### LINKED ISSUE #10597: Edit profile form field validation and password visibility
**Please describe the desired behavior.**

This issue is related with #5361 . Feature requests in the edit profile page-
1. Show password icon for "Current Password" field.
2. Real-time form field validation ( code for the same was added but due to some errors it had to be deleted, #6331 talks about the bug and gives an alternate solution which can be implemented )

### PR DESCRIPTION
**Pull request by sakshi-2412**
Fixes #10597 

https://user-images.githubusercontent.com/85152262/147675455-2a9fa747-544d-4bce-9a54-39272968659f.mp4

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by sakshi-2412** [comment]
I had to change the ids of form fields in the edit profile page as they were giving some errors (signup form fields with same ids are being rendered on this page). I tried to check and make sure that the old ids are not used somewhere else. Still I'm a bit afraid 😶

File: app/views/users/_edit_form.html.erb [hunk 1]
```diff
@@ -33,18 +33,25 @@
 
       <div class="form-group col-md-6">
         <label for="email"><%= translation('users._form.email') %></label>
-        <%= f.text_field :email, { tabindex: 3, required: true, placeholder: "you@email.com", class: 'form-control', id: 'email'} %>
+        <%= f.text_field :email, { tabindex: 3, required: true, placeholder: "you@email.com", class: 'form-control', id: 'new_email'} %>
```
Comment (TildaDares) [inline app/views/users/_edit_form.html.erb:36]: Hi @sakshi-2412, amazing work! I just wanted to know why you changed the email field's id.

**Comment by TildaDares** [comment]
@sakshi-2412 Everything seems to be working fine. Amazing work!

**Comment by sakshi-2412** [comment]
thanks :)

**Comment by jywarren** [comment]
Great work! Thanks @sakshi-2412 and thanks everyone!!!

**Pull request merged**